src/index.js

```diff
@@ -99,6 +120,11 @@ const action = argv.install             ? ACTIONS.INSTALL :
                argv['complete-install'] ? ACTIONS.COMPLETE :
                undefined;
 
+if (mode.name === 'satellite' && action) {
```

Comment:
I mostly did this because I don't know what they're for or why I'd need them and to avoid too much code churn in those other paths.  Open to thoughts on whether this is something important.